showcase/src/app/news/components/news-item/news-item.component.html

```diff
@@ -0,0 +1,22 @@
+<!--<mat-card class="news-item">-->
+	<!--<div class="mat-card-title" ng-transclude="newsTitle"></div>-->
+	<!--<div class="mat-card-subtitle" ng-transclude="newsSubtitle"></div>-->
+	<!--<div class="mat-card-content" ng-transclude="newsContent"></div>-->
+<!--</mat-card>-->
+
+<mat-card class="news-item">
+	<div class="mat-card-title">
+		<h2><ng-content select=".news-item-title"></ng-content></h2>
+	</div>
+	<div class="mat-card-subtitle">
+		<h4>Release: <ng-content select=".news-item-stark-release"></ng-content>
+			Date: <ng-content select=".news-item-date"></ng-content>
+		</h4>
+	</div>
+	<div class="mat-card-content">
+		<ng-content select=".news-item-content"></ng-content>
+	</div>
+	<div mat-card-image class="mat-content-image">
+		<ng-content select=".news-item-image"></ng-content>
```

Comment:
I'm wondering whether having a transcluded image will have an added value. What if the developer wants to add more than one image? Wouldn't it be better to add the image as part of the content so the developer can add whatever he needs?